ansible_base/lib/dynamic_config/settings_logic.py

```diff
@@ -61,6 +61,7 @@ def get_dab_settings(
             'DESCRIPTION': 'Open API',
             'VERSION': 'v1',
             'SCHEMA_PATH_PREFIX': '/api/v1/',
+            'COMPONENT_NO_READ_ONLY_REQUIRED': True,
```

Comment:
I am writing this assuming the default is False.

I don't have a problem if we want to change this to True here but we should make sure no other services are using this app yet or we will end up changing their behavior. 
If no one is using it, we can approve and merge this.
If someone is using it I am fine with changing it if we have consensus with other teams using it.
If we don't have consensus, we should default this to what it would be (again presumably False) and then individual services could override it to True in their own settings.

Reply:
Yeah, default is False.  I'll post in the DAB channel and give people a chance to complain..

Reply:
https://github.com/ansible/eda-server/blob/85cfadcf8846473e3a5a700c81108f7acd8af400/src/aap_eda/settings/default.py#L565-L576

Re-stating, what @john-westcott-iv is saying is that `SPECTACULAR_SETTINGS` in eda-server will be modified by this patch. What consequences that'll have, we don't really know.

Likewise for galaxy_ng

https://github.com/ansible/galaxy_ng/blob/b525b5f1d030b3d16c89212e92eb975958439685/galaxy_ng/app/settings.py#L229

AWX is using an older schema generator library, so not affected.

Reply:
Yeah, that is true for sure.  I asked on the DAB channel for some feedback, but it's also fine to just make this change in gateway and let other DAB users deal with it as they desire or don't desire.

Reply:
Yeah I'd like some confirmation from galaxy and eda before I +1 this. Chances are they are just facing the same bug we are solving and they probably want this fix too, but we need to check and not assume.

Reply:
@rochacbruno, @Alex-Izquierdo, @jneedle for eyes.

Reply:
+1 galaxy is not using api_documentation app, so not affected.

Reply:
Hi @Alex-Izquierdo I am trying to test w/ EDA, just having troubles getting it started.. :)

Can you clarify if you aren't using openapi spec gen at all vs you are using spectacular but without the (minimal) extra config that api_documentation brings?  But yes, you are correct that your SPECTACULAR_SETTINGS will not be modified by DAB if you aren't including the api_documentation app.

Reply:
Got EDA up and running and answered my own question.  So yes you do use spectacular, cool and I see that your solution takes care of the issue this is trying to fix, so it doesn't benefit or hurt you at all..